Jenkinsfile

```diff
@@ -1,6 +1,7 @@
 #!groovy
 
 node {
+    deleteDir()
```

Comment:
This ensures that the workspace is empty before the build runs and that each build starts from a clean slate. Ideally it would be much better if Jenkins builds work liked Travis where every build starts from a fresh slate except for files which have been explicitly saved/restored to/from a cache.